### DISCUSSION TITLE
🌌 Night Zotero 7 ALPHA Release 0.1.0-1

### DISCUSSION THREAD

**Discussion by tefkah** [tefkah/zotero-night#139] [Announcements]
# Changelog

* chore: add pnpm-lock.yaml (e6431d4)
* feat: add editor theming, sorta (eb83bc5)
* fix(#138): be compatible with new reader layout (81b10b7)

<hr /><em>This discussion was created from the release <a href='https://github.com/tefkah/zotero-night/releases/tag/0.1.0-1'>🌌 Night Zotero 7 ALPHA Release 0.1.0-1</a>.</em>

**Comment by PenTurDucKenLinE** [comment]
EXCELLENT!!! Thank you so much for the update @tefkah !!!

**Comment by toineenzo** [comment]
Awesome! Thank you for your work <3
(Ook de groetjes uit Utreg ;))

**Comment by MuiseDestiny** [comment]
Great!

**Comment by llity** [comment]
Awesome :)
